src/Projections/Reunion/Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.IDispatcherQueue.cs

```diff
@@ -0,0 +1,38 @@
+#if NET5_0_OR_GREATER
+
+#pragma warning disable CS0649
+
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.System
+{
+    /// <inheritdoc/>
+    public partial class DispatcherQueueSynchronizationContext
+    {
+        /// <summary>
+        /// A struct mapping the native WinRT <c>IDispatcherQueue</c> interface.
+        /// </summary>
+        internal unsafe struct IDispatcherQueue
+        {
+            /// <summary>
+            /// The vtable pointer for the current instance.
+            /// </summary>
+            private readonly void** lpVtbl;
```

Comment:
>lpVtbl [](http://example.com/codeflow?start=36&length=6)

Not sure about coding conventions around here, but can we remove all the hungarian notation from these files?

Reply:
Sorry, force of habit with the vtable pointers 😄
Fixed in 17ebd48dc3ecb2fb01b1235135a23bf19074f354.